plugins/CoreHome/angularjs/period-selector/period-selector.controller.js

```diff
@@ -8,9 +8,9 @@
 (function () {
     angular.module('piwikApp').controller('PeriodSelectorController', PeriodSelectorController);
 
-    PeriodSelectorController.$inject = ['piwik', '$location', 'piwikPeriods'];
+    PeriodSelectorController.$inject = ['piwik', '$location', 'piwikPeriods', 'piwikApi'];
 
-    function PeriodSelectorController(piwik, $location, piwikPeriods) {
+    function PeriodSelectorController(piwik, $location, piwikPeriods, piwikApi) {
```

Comment:
Don't need `piwikApi` here, correct?

Reply:
Hm, yes I think you are correct. I don't remember why I added this 🤷‍♂️